v1.3 RC omnibus (#2104)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-properties/config.md

</TabItem>

<TabItem value="sources">

<VersionBlock lastVersion="1.0">

> Support for the `config` property on sources was added in dbt Core v1.1

Reviewer: switch to active voice
```suggestion
We have added support for the `config` property on sources in dbt Core v1.1
```

</VersionBlock>

<VersionBlock firstVersion="1.1">

<File name='models/<filename>.yml'>

```yml
version: 2

sources:
  - name: <source_name>
    config:
      [<source_config>](source-configs): <config_value>
    tables:
      - name: <table_name>
        config:
          [<source_config>](source-configs): <config_value>
```

</File>

</VersionBlock>

</TabItem>

<TabItem value="metrics">

<VersionBlock lastVersion="1.2">

> Support for the `config` property on metrics was added in dbt Core v1.3

Reviewer: active voice 
```suggestion
We have added support for the `config` property on sources in dbt Core v1.3
```
